styles/Editor.css

```diff
@@ -939,6 +939,44 @@
   font-size: 75%;
 }
 
+.ProseMirror .table-footer {
+  font-size: 14px;
+}
+.ProseMirror .table-footer p {
+  margin-bottom: 0px;
+  margin-top: 0px;
+}
+
+.ProseMirror .table-footer  {
+  font-size: 14px;
+  padding-top: 8px;
+}
+
+.ProseMirror .table-footer p {
```

Comment:
Duplicated.

Reply:
done

Reply:
I see this is still here. it's on my branch already. you will see it when I merge my branch.

Reply:
Yes, but I added 8px padding to accommodate space for general notes or footnote elements. However, I had to remove the following code from your branch:


.ProseMirror .table-footer .footnote-element:first-child {
  padding-top: 8px;
}
This change was necessary to make it applicable to all types of footnotes, not just the first child. It addresses a more general case

Reply:
I'm talking bout the selector `.table-footer p` here. Not the `.ProseMirror .table-footer`